debezium-scripting/debezium-scripting/src/test/resources/wasm/go/go.mod

```diff
@@ -0,0 +1,5 @@
+module example.com/debezium/filter
+
+go 1.22.7
+
+require github.com/andreaTP/debezium-smt-go-pdk v0.0.0-20241126170313-176d969fecf0 // indirect
```

Comment:
The repo is in a primitive state:
https://github.com/andreaTP/debezium-smt-go-pdk

We need proper testing, structure, and possibly, the review of a Go expert.